modules/system/lang/vn/validation.php

```diff
@@ -0,0 +1,121 @@
+    'ipv6'                 => ':attribute phải là địa chỉ IPv6 hợp lệ.',
+    'json'                 => ':attribute phải là chuỗi JSON hợp lệ.',
+    'max'                  => [
+        'numeric' => ':attribute không được lơn hơn :max.',
+        'file'    => ':attribute không được lơn hơn :max kilobytes.',
+        'string'  => ':attribute không được lơn hơn :max kí tự.',
+        'array'   => ':attribute không được lơn hơn :max items.',
+    ],
+    'mimes'                => 'Loại file của :attribute phải là: :values.',
+    'mimetypes'            => 'Loại file của :attribute phải là: :values.',
+    'min'                  => [
+        'numeric' => ':attribute phải lớn hơn :min.',
+        'file'    => ':attribute phải lớn hơn :min kilobytes.',
+        'string'  => ':attribute phải nhiều hơn :min kí tự.',
+        'array'   => ':attribute phải có nhiều hơn :min item.',
+    ],
+    'not_in'               => 'Mục đã chọn :attribute không hợp lệ.',
+    'numeric'              => ':attribute phải là số.',
+    'present'              => 'Phải có trường :attribute.',
+    'regex'                => 'Định dạng của :attribute không hợp lệ.',
+    'required'             => 'Trường :attribute là bắt buộc.',
+    'required_if'          => 'Trường :attribute là bắt buộc khi :other là :value.',
+    'required_unless'      => 'Trường :attribute là bắt buộc trừ khi :other là :values.',
+    'required_with'        => 'Trường :attribute là bắt buộc khi có :values.',
+    'required_with_all'    => 'Trường :attribute là bắt buộc khi có :values.',
+    'required_without'     => 'Trường :attribute là bắt buộc khi không có :values.',
+    'required_without_all' => 'Trường :attribute là bắt buộc khi không có :values.',
+    'same'                 => ':attribute phải trùng khớp với :other.',
+    'size'                 => [
+        'numeric' => ':attribute phải có kích thước :size.',
+        'file'    => ':attribute phải có kích thước :size kilobytes.',
+        'string'  => ':attribute phải có :size kí tự.',
+        'array'   => ':attribute phải có :size items.',
+    ],
+    'string'               => ':attribute phải là chuỗi.',
+    'timezone'             => ':attribute phải là múi là hợp lệ.',
+    'unique'               => ':attribute đã tồn tại.',
+    'uploaded'             => ':attribute bị lỗi khi tải lên.',
+    'url'                  => ':attribute sai định dạng.',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Custom Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify custom validation messages for attributes using the
+    | convention "attribute.rule" to name the lines. This makes it quick to
+    | specify a specific custom language line for a given attribute rule.
+    |
+    */
+
+    'custom' => [
+        'attribute-name' => [
+            'Tên_key' => 'giá_trị',
```

Comment:
Please don't add project specific custom attributes in the default language files; this is something that should be handled at a project level by making the changes in lang/system/vn/validation.php

Reply:
Still need to remove this

Reply:
Still need to remove this :)

Reply:
done :)